homework/App.java

```diff
@@ -0,0 +1,33 @@
+package homework;
+public class App {
+
+    public static void main(String[] args) {
+    System.out.println("The Area of a 5 by 5 square is ");
+    int a = 5;
+    int b = 9;
+    int c = 2;
+    double d = 3.5;
+    int e = 3;
+    int f = 2;
+    double g = 10.2;
+    int h = 15;
+    double i = 9.0;
+    double j = 15.0;
+    double k = 27.0;
+    double l = 2.0;
+    int m = 10;
+    System.out.println((int)a + b); // Output: 14
+    System.out.println((float)l * d); // Output: 7.0
+    System.out.println((float)i / e); // Output: 3.0
+    System.out.println((float)f + a * g); // Output: 53.0
+    System.out.println((int)h / e * a); // Output: 25
+    System.out.println((float)j / (e * a)); // Output: 1.0
+    System.out.println((float)k / i / e); // Output: 1.0
+    System.out.println((float)k / (i / e)); // Output: 9.0
+    System.out.println((float)k / i / e); // Output: 1.0
```

Comment:
Again, good on you for understanding casting.  However, casting forces a particular data type.  The goal of the assignment to hypothesize what the output should be and validate your understanding by comparing what you thought would happen and what actually happened in the output.

Reply:
I challenge you to do the same redo the following equation # 4:
`2 + 5 * 10.2` 

NO casting and provide I asked in exercise 2:

> You will:
> 1. Replicate each equation using code by choosing a data type, creating a variable with that data type, assigning it a value.
>2. Write in the comments above each equation you have coded why you used the data type you have chosen.
>3. Before you run the code, make an assumption about what you expected, and record that expectation.
>4. Run the code, observe the output, and write in the Pull Request comment section the result of the output for each equation, the resulting data type, and why the result is the data type it is. If the result is different from your expectation, why do you think you made that assumption. Why do you think you received the result you did?